src/neo-vm/OperandSizeAttribute.cs

```diff
@@ -7,5 +7,17 @@ public class OperandSizeAttribute : Attribute
     {
         public int Size { get; set; }
         public int SizePrefix { get; set; }
+
+        public int GetOperandSize(byte[] script, int ip)
```

Comment:
The efficiency of array query is much higher than that of method call.

Reply:
Removed